triagebot.toml

```
@@ -642,6 +661,51 @@ cc = ["@nnethercote"]
 message = "Changes to the size of AST and/or HIR nodes."
 cc = ["@nnethercote"]
 
+[mentions."compiler/rustc_symbol_mangling/src/typeid"]
+cc = [" @rust-lang/project-exploit-mitigations", "@rcvalle"]
```

```suggestion
cc = ["@rust-lang/project-exploit-mitigations", "@rcvalle"]
```

There's a space here 😅

---

Fixed 😅. Thanks a lot, Michael!